File: app/src/main/java/com/deucat/kartik/trainbrain/MainWork/MainFragment.java
package com.deucat.kartik.trainbrain.MainWork;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentTransaction;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.FrameLayout;

import com.deucat.kartik.trainbrain.GetFragment;
import com.deucat.kartik.trainbrain.R;

public class MainFragment extends Fragment {


    public MainFragment() {}


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_main, container, false);

        Fragment initialFragment = new GetFragment();
        FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
        fragmentTransaction.replace(R.id.mainFullFragment, initialFragment);
        fragmentTransaction.commit();

        return view;
    }

}

File: app/src/main/java/com/deucat/kartik/trainbrain/mainwork/MainFragment.kt
package com.deucat.kartik.trainbrain.mainwork


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.deucat.kartik.trainbrain.R


class MainFragment : Fragment() {


    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_main, container, false)

        val initialFragment = GetFragment()
        val fragmentTransaction = fragmentManager!!.beginTransaction()
        fragmentTransaction.replace(R.id.mainFullFragment, initialFragment)
        fragmentTransaction.commit()

        return view
    }

}
